composer.json

```diff
@@ -37,7 +37,7 @@
         "phpcs-tests": "\"vendor/bin/phpcs\" --standard=phpcs-test-ruleset.xml -s ./tests/",
         "phpcbf": "\"vendor/bin/phpcbf\"",
         "phpcbf-tests": "\"vendor/bin/phpcbf\" --standard=phpcs-test-ruleset.xml -s ./tests/",
-        "sniffs": "\"vendor/bin/phpcs\" --standard=phpcs-ruleset.xml -e",
```

Comment:
No mention why this was removed in the description.

Reply:
This is just a utility script that tells me what code formatting rules are being used. I haven't ran it in a while (maybe since a major upgrade on the formatter) and it now errors out with that. The `phpcs-ruleset.xml` is picked up automatically now.